itou/siaes/models.py

```diff
@@ -151,25 +158,31 @@ class Siae(AddressMixin):  # Do not forget the mixin!
             "la plateforme."
         ),
     )
-    # This deactivation date is not enforced and only stored for
+    # This convention end date is not enforced and only stored for
```

Comment:
Tu veux dire quoi par "_is not enforced_" ?

Reply:
C'est un vestige confusant, je vire.